.github/workflows/dotnet.yml

```diff
@@ -22,3 +22,16 @@ jobs:
         run: dotnet build --no-restore
       - name: Test
         run: dotnet test --no-build --verbosity normal
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@359bebbc29cbe6c87da6bc9ea3bc930432750108
+        with:
+          ruby-version: '2.5.1'
+          working-directory: './http_server_spec'
+      - name: Bundle Ruby
+        working-directory: './http_server_spec'
+        run: bundle install
+      - name: Run Server in background
+        run: dotnet run --project TKeazirian.HTTPServer &
+      - name: Run the Ruby Tests
+        working-directory: './http_server_spec'
+        run: rake test
```

Comment:
Is there a way to group this a little differently?  Now that you have it working, I do not think we need to see everything so granular.

```text
> Build
> Unit Tests
> Acceptance Tests
> Post Run actions.checkout@v3
> Complete Job
```
<img width="268" alt="image" src="https://user-images.githubusercontent.com/26009042/172669335-0f995e63-40fd-4ce7-a1e6-396ab825fc27.png">

Reply:
Check it out now!  Should be more condensed.

Reply:
Looks Great!